lib/modules/platform/bitbucket/index.ts

```diff
@@ -945,16 +946,66 @@ export async function createPr({
           },
         )
       ).body;
-      const pr = utils.prInfo(prRes);
+      pr = utils.prInfo(prRes);
       await BitbucketPrCache.addPr(
         bitbucketHttp,
         config.repository,
         renovateUserUuid,
         pr,
       );
-      return pr;
     }
   }
+
+  logger.debug(
+    {
+      repository: config.repository,
+      title,
+      base,
+      bbAutoResolvePrTasks: platformPrOptions?.bbAutoResolvePrTasks,
+    },
+    'Auto resolve PR tasks',
+  );
```

Comment:
log only if it's true. Also you don't need to log the repository as it's already logged in metadata